lib/js-file.js

```diff
@@ -12,24 +12,37 @@ var KEYWORD_OPERATORS = {
  * File representation for JSCS.
  *
  * @name JsFile
+ * @param {Object} options
+ * @param {String} options.filename
+ * @param {String} options.source
+ * @param {Object} options.esprima
+ * @param {Object} [options.esprimaOptions]
+ * @param {Boolean} [options.es3]
+ * @param {Boolean} [options.es6]
  */
-var JsFile = function(filename, source, tree, options) {
+var JsFile = function(options) {
```

Comment:
Forgive the intrusion, but `source` is definitely not optional and should probably remain an independent parameter rather than entering the `options` object.